cmd/genji/shell/shell.go

```diff
@@ -217,32 +218,12 @@ func (sh *Shell) executeInput(in string) error {
 
 func (sh *Shell) runCommand(in string) error {
 	cmd := strings.Fields(in)
-	switch cmd[0] {
```

Comment:
I'm not sure why we don't keep this simple switch instead of doing globals and command types. Do you have a particular reason for pushing this new system?
I think we should focus on keeping things very simple.